lib/nosedrum/storage.ex

```diff
@@ -140,6 +144,35 @@ defmodule Nosedrum.Storage do
     Nostrum.Api.create_interaction_response(interaction, res)
   end
 
+  @doc """
+  Edits an interaction with a follow up response. The response is obtained by running the given
+  function/MFA tuple.
```

Comment:
Let's split this onto two lines - we don't need the implementation detail directly at the top:

```suggestion
  Edits an interaction with a follow up response.
  
  The response is obtained by running the given function/MFA tuple, see `c:callback/0`.
```